# Систематики сопряжённые с катами

In [51]:
import numpy as np
energy = [919.966, 929.985, 930.481, 934.705, 935.478, 936.305, 937.505, 938.176, 938.323, 938.723, 938.887, 939.467, 939.594, 940.166, 940.207, 940.608, 940.96, 941.882, 941.995, 943.495, 944.817, 944.993, 945.461, 947.5, 949.995, 950.15, 950.29, 950.359, 950.945, 952.012, 953.003, 953.997, 962.687, 968.805, 970.583, 972.917, 977.158, 980.248, 987.81, 990.293, 1003.5]
init_xsec = [0.0006, 0.0032, -0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.1865,0.7173,0.7517,0.8842,0.9152,0.7023,0.7781,0.7077,0.7389]
init_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 0.3483, 0.0701, 0.058, 0.0991, 0.3042, 0.0396, 0.1105, 0.044, 0.007]

energy = np.array(energy)
init_xsec = np.array(init_xsec)
init_xsec_err = np.array(init_xsec_err)

def calc_systematic(coll_xsec: np.ndarray):
    delta = np.round(abs(init_xsec - coll_xsec)/init_xsec * 100, 4)
    val = np.max(delta[delta!=0 & (energy>965)])
    return val


0.8202111111111111

## Коллинеарные события

* [x] Отбор на коллинеарность:
max_delta_phi: 0.15 -> 0.2
max_delta_theta: 0.2 -> 0.25

In [41]:
import numpy as np

coll_loose_xsec = [0.0006, 0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.1876,0.7181,0.7516,0.8847,0.9169,0.7046,0.7791,0.709,0.7401]
coll_loose_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 1.0846, 0.0556, 0.0495, 0.1021, 0.3927, 0.0365, 0.0956, 0.0386, 0.0069]

coll_loose_xsec = np.array(coll_loose_xsec)
coll_loose_xsec_err = np.array(coll_loose_xsec_err)


calc_systematic(coll_loose_xsec)

0.3275

max_delta_phi:0.15->0.1
max_delta_theta:0.2->0.15

In [45]:
coll_tightened_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.1889,0.7145,0.747,0.8739,0.9131,0.6987,0.7732,0.706,0.7378]
coll_tightened_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 1.0864, 0.0553, 0.0492, 0.1009, 0.391, 0.0362, 0.095, 0.0385, 0.0071]

coll_tightened_xsec = np.array(coll_tightened_xsec)
coll_tightened_xsec_err = np.array(coll_tightened_xsec_err)

print(calc_systematic(coll_tightened_xsec))

1.1649


* [x] Отбор на максимальный rho трека:
max_track_rho: 0.2 -> 0.3
               0.2 -> 0.15

In [47]:
coll_loose_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.0928,0.6943,0.7298,0.8464,0.8913,0.6917,0.7594,0.6998,0.7322]
coll_loose_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 0.9862, 0.0538, 0.048, 0.0975, 0.3799, 0.0358, 0.0933, 0.0382, 0.0069]

coll_loose_xsec = np.array(coll_loose_xsec)
coll_loose_xsec_err = np.array(coll_loose_xsec_err)

coll_tightened_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.2672,0.7409,0.7748,0.9266,0.9606,0.7124,0.8015,0.7165,0.746]
coll_tightened_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 1.1689, 0.0574, 0.051, 0.1072, 0.4137, 0.0368, 0.0983, 0.039, 0.0071]

coll_tightened_xsec = np.array(coll_tightened_xsec)
coll_tightened_xsec_err = np.array(coll_tightened_xsec_err)

print(calc_systematic(coll_tightened_xsec))
print(calc_systematic(coll_loose_xsec))

6.8015
7.8972


* [x] Отбор на отношение импульсов:
max_mom_ratio: 0.1 -> 0.075
               0.1 -> 0.15

In [48]:
coll_loose_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.1877,0.7163,0.7504,0.8834,0.9105,0.7019,0.7782,0.707,0.7378]
coll_loose_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 1.0861, 0.0554, 0.0494, 0.102, 0.3909, 0.0363, 0.0955, 0.0385, 0.0071]

coll_loose_xsec = np.array(coll_loose_xsec)
coll_loose_xsec_err = np.array(coll_loose_xsec_err)

coll_tightened_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.1744,0.7183,0.7516,0.8889,0.9096,0.7048,0.7819,0.7104,0.7394]
coll_tightened_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 1.07, 0.0556, 0.0494, 0.1025, 0.3889, 0.0365, 0.096, 0.0387, 0.0069]


coll_tightened_xsec = np.array(coll_tightened_xsec)
coll_tightened_xsec_err = np.array(coll_tightened_xsec_err)

print(calc_systematic(coll_tightened_xsec))
print(calc_systematic(coll_loose_xsec))

1.0198
0.5135


* [ ] Пределы фитирования угловых распределений для определения Ge/Gm:
theta_cut: 0.8 -> 0.6

In [49]:
coll_tightened_xsec = [0.0006,0.0032,-0.0003,0.0027,0.0014,0.0013,0.0028,0.0119,0.1739,0.2952,0.3686,0.4504,0.4797,0.5205,0.5159,0.573,0.5487,0.591,0.6168,0.6342,0.7676,0.6638,0.7411,0.7686,0.7653,0.9511,0.8658,1.0661,0.8032,0.7592,0.7973,0.6382,1.0749,0.7179,0.7641,0.7854,0.816,0.709,0.7225,0.7051,0.7389]
coll_tightened_xsec_err = [0.0211, 0.0213, 0.0227, 0.0213, 0.0233, 0.0227, 0.0212, 0.024, 0.0194, 0.0602, 0.0284, 0.0691, 0.0309, 0.0726, 0.0312, 0.0764, 0.0329, 0.0941, 0.0346, 0.0343, 0.0762, 0.0337, 0.0386, 0.0355, 0.0754, 0.1491, 0.0616, 0.1907, 0.3152, 0.1013, 0.2054, 0.1667, 0.3844, 0.1017, 0.0905, 0.1252, 0.4341, 0.068, 0.1469, 0.0712, 0.007]
coll_tightened_xsec = np.array(coll_tightened_xsec)
coll_tightened_xsec_err = np.array(coll_tightened_xsec_err)

print(calc_systematic(coll_tightened_xsec))

11.1739


У коллинеарных систематика от отборов: 10%
От Ge/Gm: 11%

## Аннигиляционные события

* [ ] Отбор на максимальный кластер, пришитый к треку в событии:
Tracks’s Energy depo: 400 -> 300

* [ ] Отбор на полное энерговыделение:
tot_cal_depo: 500 -> 600
              500 -> 450

* [ ] Пределы сигнальной области: